automated-comments/go/hamming/minimize_slice_of_rune_conversions.md

```diff
@@ -0,0 +1,3 @@
+You are converting the variable `%{name}` multiple times to a `[]rune`. 
+Since this is a comparatively expensive operation it would be better to do the conversion once, save the result
+to a variable and reuse that.
```

Comment:
Do we know that they've used the name `name`?

To tighten it up, perhaps:

> Prefer to convert to a `[]rune` only once, saving the result in a variable and reusing it, as this conversion is a comparatively expensive operation.

Reply:
With the used variable name maybe:

> Prefer to convert `%{name}` to a `[]rune` only once, saving the result in a variable and reusing it. This conversion is a comparatively expensive operation.